nengo_gui/components/netgraph.py

```diff
@@ -453,7 +469,4 @@ def create_connection(self, client, conn, parent):
         info = dict(uid=uid, pre=pres, post=posts, type='conn', parent=parent)
         client.write(json.dumps(info))
 
-
-class NetGraphTemplate(Template):
-    cls = NetGraph
-    config_params = dict()
+NetGraphTemplate = NetGraph
```

Comment:
Yes.  I'll add a comment to make that clear (and perhaps find a way to do it in one place, rather than in each Component file).